docs/guides/aerogear-android/pipe.asciidoc

```diff
@@ -50,154 +56,107 @@ public class Developer {
     
     @RecordId
     private Long id;
+    private String name;
     private String twitter;
     private String photoURL;
 
 }
 ----    
 
-How? link:/docs/specs/aerogear-android/org/jboss/aerogear/android/Pipeline.html[*Pipeline*] class is used to instantiate and manage a link:/docs/specs/aerogear-android/org/jboss/aerogear/android/pipeline/Pipe.html[*Pipe*] object. `Pipeline.pipe` creates and returns an instance of link:/docs/specs/aerogear-android/org/jboss/aerogear/android/pipeline/Pipe.html[*Pipe*]. You don't have to keep a reference to the result. `Pipeline.get(String, Activity)` will return a instance of link:/docs/specs/aerogear-android/org/jboss/aerogear/android/pipeline/LoaderPipe.html[*LoaderPipe*] which is really a wrapper around the original link:/docs/specs/aerogear-android/org/jboss/aerogear/android/pipeline/Pipe.html[*Pipe*] object. There are also `Pipeline.get` methods for _Fragments_, _FragmentActivity_ and _Fragment_ (link:http://developer.android.com/tools/support-library[*Android's support library*]) classes 
+==== Creating a Pipe
 
-[source, java]
+link:/docs/specs/aerogear-android-pipe/org/jboss/aerogear/android/pipeline/PipeManager.html[*PipeManager*] class is used to instantiate and manage a link:/docs/specs/aerogear-android-pipe/org/jboss/aerogear/android/pipeline/Pipe.html[*Pipe*] object. You don't have to keep a reference to the result. `Pipeline.get(String, Activity)` will return a instance of link:/docs/specs/aerogear-android-pipe/org/jboss/aerogear/android/pipeline/LoaderPipe.html[*LoaderPipe*] which is really a wrapper around the original link:/docs/specs/aerogear-android-pipe/org/jboss/aerogear/android/pipeline/Pipe.html[*Pipe*] object. There are also `Pipeline.get` methods for _Fragments_ classes 
+
+[source,java]
 ----
-public class MyActivity extends ListActivity {
+PipeManager.config("developers", RestfulPipeConfiguration.class)
+                    .withUrl(new URL("http://www.server.com/developer"))
+                    .forClass(Developer.class);
 
-    private Pipeline pipeline;
+Pipe<Developer> pipe = PipeManager.getPipe("developers");
+----
 
-    protected void onCreate() {
-        String serverURL = "http://www.server.com";
-        pipeline = new Pipeline(serverURL);
-        pipeline.pipe(Developer.class);
-    }
+Here is an example of using the link:/docs/specs/aerogear-android-pipe/org/jboss/aerogear/android/pipeline/Pipe.html[*Pipe*] API to retrieve the JSON payload shown earlier. Note that the library will automatically marshall the payload to your model, leaving you only with task to display the results in the UI. Further and equal important, it will respect the Activity (or Fragment) lifecycle, a common source of problem bugging Android development as we will discuss next
```

Comment:
Further and equal*ly* important, it will respect the Activity (or Fragment) lifecycle, a common source of problem*s* bugging Android *developers* as we will discuss next